backend/app/schemas.py

```
@@ -1 +1,99 @@
+
+
+class CreateTablesResponse(BaseModel):
+    """テーブル作成レスポンス"""
+    success: bool = Field(..., description="作成成功フラグ")
+    theme: str = Field(..., description="AIが選択したテーマ")
+    message: Optional[str] = Field(None, description="ユーザー向けメッセージ")
+    tables: Optional[List[str]] = Field(None, description="作成したテーブル名リスト（将来実装）")
+    details: Optional[Dict[str, Any]] = Field(None, description="詳細情報（将来実装）")
+
+
+class GenerateProblemResponse(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)